packages/utilities/src/modalize.test.ts

```diff
@@ -143,24 +141,6 @@ describe('modalize', () => {
     expect(getHiddenElements()).toStrictEqual([]);
   });
 
-  it('does nothing in SSR', () => {
-    // can't fully simulate SSR but at least set the variable
-    setSSR(true);
-
-    document.body.innerHTML = `
-      <div id="siblingBefore"></div>
-      <div id="${modalizeId}"></div>
-      <div id="siblingAfter"></div>
-    `;
-
-    const child = document.getElementById(modalizeId)!;
-    const unmodalize = modalize(child);
-    expect(getHiddenElements()).toStrictEqual([]);
-
-    unmodalize();
-    expect(getHiddenElements()).toStrictEqual([]);
-  });
-
```

Comment:
This had to be removed since we are not faking SSR anymore, but can be fixed by moving into ssr-tests.

Reply:
This one could be problem, in theory we are already testing it due to `modalize` using `getDocument/getWindow`. The problem is that the original tests walks the tree and finds hidden elements, but this is not possible and shouldn't even be testes anymore because we know getDocument/getWindow will return undefined (therefore there's nothing to walk).